tests/fields/fields_list.test.ts

```diff
@@ -0,0 +1,108 @@
+        test.concurrent('has no doubles with the same "path"', async () => {
+            const matches = fields.filter(fields_item => fields_item.path === field.path);
+            expect(matches).toBeArrayOfSize(1);
+        });
+
+        test.concurrent(`should ${non_fillable_fields[group_name].indexOf(field.path) === -1 ? 'be fillable by at least one source' : 'not be fillable by any source'}`, async () => {
+            if (field.fillable_by.length > 0) {
+                expect(non_fillable_fields[group_name]).not.toContain(field.path);
+            } else {
+                expect(non_fillable_fields[group_name]).toContain(field.path);
+            }
+        });
+
+        const specs_field_path: string[] = [];
+        const report_field_path: string[] = [];
+
+        field.path.split('.').forEach(part => {
+            specs_field_path.push('properties');
+            if (part.endsWith('[]')) {
+                specs_field_path.push(...[part.replace('[]', ''), 'items']);
+                // If path contains '...[]' - work directly with first item inside report item array
+                report_field_path.push(part.replace('[]', '.0'));
+            } else {
+                specs_field_path.push(part);
+                report_field_path.push(part);
+            }
+        });
+
+        test.concurrent(`should have declaration in "reports/${group_name}/json-schema.json"`, async () => {
+            expect(get(report_schema, specs_field_path.join('.'))).toBeObject();
+        });
+
+        test.concurrent(`should have same description in "reports/${group_name}/json-schema.json"`, async () => {
+            expect(field.description).toInclude(get(report_schema, specs_field_path.join('.') + '.description'));
+        })
+
+        test.concurrent(`should have identical set of values in "fillable_by" property with "reports/${group_name}/json-schema.json"`, async () => {
+            expect(get(report_schema, specs_field_path.join('.') + '.fillable_by')).toIncludeSameMembers(field.fillable_by);
+        });
+
+        test.concurrent.each(examples_reports_test_cases)(`should presents in %s and have same type`, async (report_path, report_content) => {
+            const report_field_value = get(report_content, report_field_path.join('.'), "__NOT_SETTED__");
+
+            expect(report_field_value).not.toBe('__NOT_SETTED__');
+
+            field.types.forEach(field_type => {
+                if (report_field_value !== null) {
+                    switch (field_type) {
+                        case 'object':
+                            expect(report_field_value).toBeObject();
+                            break;
+                        case 'array':
+                            expect(report_field_value).toBeArray();
+                            break;
+                        case 'boolean':
+                            expect(report_field_value).toBeBoolean();
+                            break;
+                        case 'float':
```

Comment:
@tarampampam 
In this case we can't detect real data type (`float` or `integer`) for values such as `100.0`, e.g 
```js
Number.isInteger(100.0) // true
parseFloat(100.0) // 100
```
 Also json-schema has no data type `float`. What will be better solution for this case?

Reply:
Seems like not :(

Reply:
Made some changes:

```js
case 'float':
    expect(report_field_value).toBeNumber();
    break;
case 'integer':
    expect(Number.isInteger(report_field_value)).toBeTrue();
    break;
```
Now we can be sure that values like `1.3` doesn't pass test for `integer` type at least:

```js
Number.isInteger(1.3) // false
// but
Number.isInteger(1.0) // true =(
```